apache-rat-gradle/src/site/apt/index.apt

```diff
@@ -66,6 +66,17 @@ rat {
     // Fail the build on rat errors, defaults to true
     failOnError = false
 
+    // Enable XML RAT output, defaults to true
+    xmlOutput = true
+
+    // Enable HTML RAT output, defaults to true
+    htmlOutput = true
+
+    // Enable plain text RAT output, defaults to false
+    // Please note that if xml or html output is enabled too,
+    // then two RAT runs will be needed to produce all reports.
```

Comment:
That's an extremely expensive requirement for large code bases.  We've seen with the RAT/gradle integration that the Samza team uses that it's definitely possible to generate multiple outputs from one run.